compilation fail for  aarch64 using alpine OS.

I am attempting to build librespot on a 64bits aarch64 Raspberry Pi using docker alpine:edge
The compilation fail with the following:

```logs

error[E0425]: cannot find value `MAP_32BIT` in crate `libc`
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/nix-0.10.0/src/sys/mman.rs:49:9
   |
49 |         MAP_32BIT;
   |         ^^^^^^^^^ not found in `libc`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0425`.
error: could not compile `nix`.
warning: build failed, waiting for other jobs to finish...
error: build failed

```

From my research the issue is fixed in nix > 0.11.0

Is it possible to update Cargo dependency to point to a later version of ‘nix’ i.e. 0.14.1 or later?

Your help would be much appreciated.

--- a/connect/Cargo.toml
+++ b/connect/Cargo.toml
@@ -35,7 +35,7 @@ aes-ctr = "0.3"
 block-modes = "0.3"
 
 dns-sd = { version = "0.1.3", optional = true }
-libmdns = { version = "0.2.4", optional = true }
+libmdns = { version = "0.2.5", optional = true }
 
 [features]
 default = ["libmdns"]

--- a/src/main.rs
+++ b/src/main.rs
@@ -51,9 +51,9 @@ fn setup_logging(verbose: bool) {
         }
         Err(_) => {
             if verbose {
-                builder.parse_filters("mdns=info,librespot=trace");
+                builder.parse_filters("libmdns=info,librespot=trace");
             } else {
-                builder.parse_filters("mdns=info,librespot=info");
+                builder.parse_filters("libmdns=info,librespot=info");
             }
             builder.init();
         }

